lib/src/bindings/dlib.dart

```diff
@@ -7,12 +7,31 @@ import 'dart:ffi';
 import 'dart:io';
 import 'dart:isolate' show Isolate;
 
-String _getPlatformSpecificName() {
+const Set<String> _supported = {'linux64'};
+
+String _detectHost() {
+  final architecture = sizeOf<IntPtr>() == 4 ? '32' : '64';
+  var os;
   if (Platform.isLinux) {
-    return 'libtensorflowlite_c-linux.so';
+    os = 'linux';
+  } else if (Platform.isMacOS) {
+    os = 'mac';
+  } else if (Platform.isWindows) {
+    os = 'windows';
+  } else {
+    throw new Exception('Unsupported platform!');
```

Comment:
Just confirming that you want to throw here (and in `_getObjectFilename()`). If so, you may want to document that via dartdoc for the method to bring it to callers attention (I'd think it would be a bit surprising).

Reply:
Good point! I'll throw in a single location and add a dartdoc.